src/content/docs/configuration/targets.mdx

```diff
@@ -5,6 +5,12 @@ sidebar:
   label: Targets
 ---
 
+import Label from '@components/Label.astro'
+
+<Label>
+  Distribution: **Open Source**, **Cloud**, **Self-Managed**
```

Comment:
Targets are only available in Open Source

Reply:
@Tpuljak there is a devcontiner builder in EE?